docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/StochasticDualCoordinateAscent.cs

```diff
@@ -1,7 +1,6 @@
-﻿using Microsoft.ML.Data;
-using Microsoft.ML.SamplesUtils;
+﻿using Microsoft.ML.SamplesUtils;
 
-namespace Microsoft.ML.Samples.Dynamic.Trainers.MulticlassClassification
```

Comment:
>.Trainers.MulticlassClassification [](start = 38, length = 34)

why are we removing this? we're using the long namespace for trainers to prevent name conflicts

Reply:
name conflicts? those are standalone examples. 

---
In reply to: [271977575](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271977575) [](ancestors = 271977575)

Reply:
I thought every class had its own distinctive name?

---
In reply to: [271977575](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271977575) [](ancestors = 271977575)

Reply:
ic, we call them the same between tasks. Reverting. 

---
In reply to: [272288927](https://github.com/dotnet/machinelearning/pull/3187#discussion_r272288927) [](ancestors = 272288927,271977575)